source/unified-test-format/unified-test-format.rst

```diff
@@ -218,7 +218,32 @@ Test runners MUST support the following types of entities:
 - Collection. See `entity_collection`_ and `Collection Operations`_
 - ClientSession. See `entity_session`_ and `Session Operations`_.
 - GridFS Bucket. See `entity_bucket`_ and `Bucket Operations`_.
-- ChangeStream. See `ChangeStream Operations`_.
+
+.. _entity_changestream:
+
+- ChangeStream. Change stream entities are special in that they are not
+  defined in `createEntities`_ but are instead created by using
+  `operation.saveResultAsEntity <operation_saveResultAsEntity_>`_ with a
+  `client_createChangeStream`_, `database_createChangeStream`_, or
+  `collection_createChangeStream`_ operation.
+  
+  Test files SHOULD NOT use a ``watch`` operation to create a change
+  stream, as the implementation of that method may vary among drivers. For
+  example, some implementations of ``watch`` immediately execute ``aggregate``
+  and construct the server-side cursor, while others may defer ``aggregate``
+  until the change stream object is iterated.
+
+  See `Cursor Operations`_ for a list of operations.
```

Comment:
I renamed `ChangeStream Operations` to a more generic `Cursor Operations` because it defines operations that pertain to both `ChangeStream` and `FindCursor` entities. Because of that, information that only applies to change streams (e.g. the guidance about not using a `watch` operation) had to be moved here.